docs/Resonance_Compensation.md

```diff
@@ -150,16 +143,16 @@ For most of the printers, either MZV or EI shapers can be recommended. This
 section describes a testing process to choose between them, and figure out
 a few other related parameters.
 
-Print the ringing test model as follows (assuming you already have
-shaper_freq_x/y set and max_accel/max_accel_to_decel increased to 7000 in
-printer.cfg file):
+Print the ringing test model as follows:
 
-1. Restart the firmware: `RESTART`.
-2. Disable Pressure Advance: `SET_PRESSURE_ADVANCE ADVANCE=0`.
-3. Execute `SET_INPUT_SHAPER SHAPER_TYPE=MZV`.
-4. Execute the command
-   `TUNING_TOWER COMMAND=SET_VELOCITY_LIMIT PARAMETER=ACCEL START=1250 FACTOR=100 BAND=5`.
-5. Print the test model sliced with the suggested parameters.
+1. Restart the firmware: `RESTART`
+2. Prepare for test:
+   `SET_VELOCITY_LIMIT ACCEL_TO_DECEL=7000 SQUARE_CORNER_VELOCITY=5`
```

Comment:
Following the doc, this printed model will also be used later to choose max_accel. Therefore, I would suggest not to override SQUARE_CORNER_VELOCITY at least for this model (and maybe, not anywhere in the testing via this command, see my other comment above), as the results will be different from the real-life printing.